packages/google_maps_flutter/google_maps_flutter_web/lib/src/google_maps_flutter_web.dart

```
@@ -132,6 +132,15 @@ class GoogleMapsPlugin extends GoogleMapsFlutterPlatform {
     return moveCamera(cameraUpdate, mapId: mapId);
   }
 
+  @override
+  Future<void> animateCameraWithConfiguration(
+    CameraUpdate cameraUpdate, {
+    required CameraUpdateAnimationConfiguration configuration,
+    required int mapId,
+  }) {
+    return animateCamera(cameraUpdate, mapId: mapId);
```

This will need a TODO referencing an issue, as with the test disabling.

---

Unfortunately, I don't think Google Maps on the web supports *any* control of the duration of the animation... See docs for `moveCamera`, `panTo` and friends:

* https://developers.google.com/maps/documentation/javascript/reference/map#Map.moveCamera

So I think you can remove the TODO, the link and everything. Maybe we can log that the value will be silently ignored to the js debug console to let users know? Or at least document in the method of the plugin that the value will not be used on the web.

---

Discussion about this on web-related issue here: https://github.com/flutter/flutter/issues/159265#issuecomment-2580116988

---

Web implementation is not necessary as platform interface now automatically defaults to 
`animateCamera`
https://github.com/flutter/packages/pull/7648/files#diff-82107f54c1af7174644137b415b811aa06290c54af28a3a81de0589ca9915e63R207
I will remove web-implementation from this PR as there is separate issue for web impementation: https://github.com/flutter/flutter/issues/159265